person.rb

```diff
@@ -29,4 +32,9 @@ def of_age?
 
     false
   end
+
+  def add_rentals(date)
+    rental = Rental.new(date)
+    @rentals << rental
+  end
```

Comment:
- Similar to my suggestion for the `book` class, please implement properly the `has-many` relationship between the `person` class and the `rentals` class 🙏